src/coprocessor/checksum.rs

```diff
@@ -70,11 +70,7 @@ impl<S: Snapshot> ChecksumContext<S> {
     }
 
     fn new_scanner(&self, range: KeyRange) -> Result<Scanner<SnapshotStore<S>>> {
-        let scan_on = match self.req.get_scan_on() {
-            ChecksumScanOn::Table => ScanOn::Table,
-            ChecksumScanOn::Index => ScanOn::Index,
-        };
-        Scanner::new(&self.store, scan_on, false, false, range).map_err(Error::from)
+        Scanner::new(&self.store, false, false, range).map_err(Error::from)
```

Comment:
So here we can not collect the number of `tablescan` or `indexscan` for checksum?

Reply:
Sorry, Seems we do not collect them before.

Reply:
The number is only used for TiDB to output slow log. No need for checksum.